Documentation/io.mender.Update1.xml

```diff
@@ -6,6 +6,8 @@
     io.mender.Update1:
     @short_description: Mender Update Management API v1
 
+    !! This feature will be removed in Mender client 4.0. For server-side support duration, please refer to our [blog post](https://mender.io/blog/mender-3-6-auto-generation-of-delta-updates#:~:text=Deprecation%3A%20Synchronized%20updates).
```

Comment:
@TheMeaningfulEngineer: This file has been deleted in the `feature-c++-client` branch, which means that this will be deleted too as soon as we merge. Was that perhaps a mistake..?

Reply:
I think this should remain in the c++ branch for the sake of documenting the deprecation.
Would remove it only once the go client isn't part of any currently supported release bundle any more.

Reply:
Ok, I will restore the file in our branch, which should then merge cleanly with this change.

In the meantime, merging here.